PxGet/Functions/Invoke-PxGet.ps1

```diff
@@ -26,7 +26,7 @@ function Invoke-PxGet
     $DebugPreference = 'Continue'

```

Comment:
The user should be able to change the path where pxget installs the module. Add support for a "Path" property in the pxget.json file for each module, and pxget uses that path as the path to a directory where the modules should be saved.